test/crdsvalidation/konnect.konghq.com/konnectgatewaycontrolplane_test.go

```diff
@@ -116,6 +116,7 @@ func TestKonnectGatewayControlPlane(t *testing.T) {
 		common.TestCasesGroup[*konnectv1alpha1.KonnectGatewayControlPlane]{
 			{
 				Name: "konnect.authRef change is not allowed for Programmed=True",
+				SkipReason: "Fails when v1alpha2 is the storage version due to missing conversion logic. This test could be re-enabled by implementing conversion logic using a conversion webhook",
```

Comment:
I see this PR states that it "Fixes: https://github.com/Kong/kubernetes-configuration/issues/385". In that case I suggest to create a separate issue to introduce said conversion webhook and link it here to the skip reason.

Reply:
At this point, we haven’t decided yet whether we’ll introduce a conversion webhook or make a breaking change, so I’ve added the skip reason to clarify the current test status and our options. Once we finalize the approach, I’ll update the tests if we go the conversion webhook route.

Also, I will remove the "fixes" reference since the original issue is still tracking the problem.